draft-ietf-dhc-dhcpv4-over-dhcpv6-ra.md

```diff
@@ -164,8 +167,13 @@ As the 4o6RA acts as a DHCP 4o6 client, all prerequisites and configuration
 that apply to the DHCP client in {{Section 5 of RFC7341}} are also applied to the 4o6RA.
 
 As the 4o6RA takes the role of the client in respect to {{RFC7341}},
-it also takes the responsibility for finding a suitable interface; that can be
-a network interface or another Relay Agent.
+it is responsible for determining a suitable interface on which to be a
+DHCPv6 client, and it is responsible for locating a suitable DHCPv6
+server or relay agent.
+   It is RECOMMENDED that the 4o6RA requests the DHCP 4o6
+   Server Address option from the server by sending the option code in
+   an Option Request option using an INFORMATION-REQUEST  as
+   described in {{draft-ietf-dhc-rfc8415bis}}.
```

Comment:
```suggestion
server or relay agent and obtain the necessary IPv6 configuration..
As specified in {{RFC7341}}, the 4o6RA, acting as 4o6 client, therefore has to request
the DHCP 4o6 Server Address option from the server by sending the
Option Request option as described in {{draft-ietf-dhc-rfc8415bis}}
before it can use the 4o6 transport.
```
It's not good practice to specify something normatively here again, which is already specified normatively somewhere else. There let's just give a reference and restate what's required by roc7341 already.